packages/jsActions/nanoflow-actions-native/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## Fixed
+
+- We fixed a timeout issue while getting the current location with minimum accuracy.
```

Comment:
I think we should also mention about the breaking change, need of `Request location permission` before using this JS action in our release notes, and documentation of this "Get current location with minimum accuracy" as we did for "Get current location".